core/src/types/operator/builder.rs

```diff
@@ -95,6 +97,22 @@ impl Operator {
         Ok(OperatorBuilder::new(acc))
     }
 
+    /// TODO: document this.
+    ///
+    /// TODO: improve those examples
+    /// # Examples
+    /// ```
+    /// let op = Operator::from_uri("fs://?root=/tmp/test", vec![])?
+    ///
+    /// Ok(())
+    /// ```
+    pub fn from_uri(
```

Comment:
should this method be called `via_uri`? 

The `from_iter` method returns an `OperatorBuilder` instead of `Operator` https://github.com/apache/opendal/blob/6eed396b7e48abf42b7548f719ee987840df1bf4/core/src/types/operator/builder.rs#L147

and the `via_iter` method returns an `Operator` (as this `from_uri` method does) https://github.com/apache/opendal/blob/6eed396b7e48abf42b7548f719ee987840df1bf4/core/src/types/operator/builder.rs#L185

Should we include two new methods? `via_uri` and `from_uri`?

Reply:
`from_uri` is good enough, all `via_xxx` API may be removed in the future.